src/constants/abi/StakingSingleReward.json

```diff
@@ -0,0 +1,319 @@
+[
+	{
+		"constant": false,
```

Comment:
the convention is to call the ABI as the contract name

Reply:
yes that's the convention